test/io.test.js

```diff
@@ -170,18 +188,24 @@ describe('IO', function(){
         describe('readLong()', function(){
             it('should decode and return a long', function(){
                 block.write(new Buffer([0x94, 0x02]));
-                decoder.readLong().should.equal(138);
+                decoder.readLong().equals(138).should.be.true;
```

Comment:
Agreed - let's not worry about the Long module for now - it seems like it has had its own issues regarding `valueOf` :stuck_out_tongue_closed_eyes: